base/system/diskpart/diskpart_msg.mc

```diff
@@ -6699,22 +6699,22 @@ Example:
     SELECT VOLUME=C:\\MountH
 .
 Language=Romanian
-    Selects the specified volume and shifts the focus to it.
+    Selectează volumul specificat și mută focalizarea pe el.
 
-Syntax:  SELECT VOLUME={<N> | <D>}
+Sintaxă:  SELECT VOLUME={<N> | <D>}
 
-    VOLUME=<N>  The number of the volume to receive the focus.
+    VOLUME=<N>  Numărul volumului care urmează să primească focalizare.
 
-    VOLUME=<D>  The drive letter or mounted folder path of the volume
-                to receive the focus.
+    VOLUME=<D>  Litera discului sau calea dosarului montat al volumului
+                ce urmează să primească focalizare.
 
-    If no volume is specified, the select command lists the current volume with
-    focus. You can specify the volume by number, drive letter, or mounted folder
-    path. On a basic disk, selecting a volume also gives the corresponding
-    partition focus. You can view the numbers of all volumes on the computer by
-    using the list volume command.
+    Dacă niciun volum nu e specificat, comanda SELECT listează volumul actual
+	cu focalizare. Puteți specififca volumul după număr, litera discului sau calea
```

Comment:
You have a typo here. Also, in Romanian it may sound more naturally to add "după" for each enumerated option.